ucfcs/Fall2022-Group20
# Julio/problem generator

This PR adds the random problem generator. The generator is implemented in flashcards for roadmap one but a refactoring of duplicate files is needed before fully implementing it. 

To test: 
Under `com.GrowthPlus(androidTest)`  folder, run MathProblemGeneratorTest, look for the double green arrow next to `public class MathProblemGeneratorTest`. 
Make sure all tests pass.
